less/mixins/grid.less

```diff
@@ -13,8 +13,10 @@
 
 // Creates a wrapper for a series of columns
 .make-row(@gutter: @grid-gutter-width) {
-  margin-left:  ceil((@gutter / -2));
-  margin-right: floor((@gutter / -2));
+  @_margin_left_helpvar:  ceil((@gutter / 2));
```

Comment:
Less's "Strict math" option is set to true in `Gruntfile.js`, which makes the behavior pretty weird (set "Strict math" to enabled in the options tab at the top of the page before trying the examples):
A. [One set of parenthesis](http://less2css.org/#%7B%22less%22%3A%22.row%20%7B%5Cn%20%20%40gutter%3A%2015px%3B%5Cn%20%20margin-left%3A%20%20-(ceil((%40gutter%20%2F%202)))%3B%5Cn%20%20margin-right%3A%20-(floor((%40gutter%20%2F%202)))%3B%5Cn%7D%22%7D) -- doesn't work
B. [Two sets of parenthesis](http://less2css.org/#%7B%22less%22%3A%22.row%20%7B%5Cn%20%20%40gutter%3A%2015px%3B%5Cn%20%20margin-left%3A%20%20-((ceil((%40gutter%20%2F%202))))%3B%5Cn%20%20margin-right%3A%20-((floor((%40gutter%20%2F%202))))%3B%5Cn%7D%22%7D) -- works

I'll change it to use double parenthesis around the `ceil()` and `floor()` functions